DiffusionPreprocessing/scripts/eddy_postproc.sh

```diff
@@ -87,13 +87,31 @@ else
 fi
 #fi
 
+
+#Create a mask representing voxels within the field of view for all volumes prior to dilation
+${FSLDIR}/bin/fslmaths ${datadir}/data -abs -Tmin -bin -fillh ${datadir}/fov_mask
```

Comment:
Maybe add a comment here about why we are using `-abs`?
e.g.,
`# 'eddy' can return negative values in some low signal locations, so use -abs for determining the fov mask`

Reply:
Done

Reply:
done